src/config/AppConfiguration.js

```diff
@@ -30,7 +30,7 @@ export const SUPPORT_URL = 'https://support.cosmotech.com';
 export const COSMOTECH_URL = 'https://cosmotech.com';
 
 // Documentation url
-export const DOCUMENTATION_URL = 'doc.pdf';
+export const DOCUMENTATION_URL = ' https://portal.cosmotech.com/resources/platform-resources/platform-help';
```

Comment:
There is a whitespace before `https`, I don't know if it could break something, but we should probably remove it